neqo-crypto/build.rs

```diff
@@ -295,6 +298,53 @@ fn build_bindings(base: &str, bindings: &Bindings, flags: &[String], gecko: bool
         .expect("couldn't write bindings");
 }
 
+fn pkg_config() -> Vec<String> {
+    let modversion = Command::new("pkg-config")
+        .args(["--modversion", "nss"])
+        .output()
+        .expect("pkg-config reports NSS as absent")
+        .stdout;
+    let modversion = String::from_utf8(modversion).expect("non-UTF8 from pkg-config");
+    let modversion = modversion.trim().to_string();
+    // The NSS version number does not follow semver numbering, because it omits the patch version
+    // when that's 0. Deal with that.
+    let modversion_for_cmp = if modversion.split('.').count() == 2 {
+        modversion.clone() + ".0"
+    } else {
+        modversion.clone()
```

Comment:
You shouldn't need to clone here.  You already have an owned copy (that's what the `to_string()` above did.  That said, I would not use `to_string()` above, instead making a copy here instead.